source/index.md

```diff
@@ -405,7 +405,9 @@ There is a single endpoint available for updating the webhook url for a publicat
 
 The set webhook endpoint provides you with access to set a publication's webhook url.
 
-To make publishing stories easier, we provide a webhook that will push stories to you as they are completed. Our webhook will submit a POST request to the URL you specifiy so that you don't have to poll for newly completed stories. The body of the request will contain the JSON data for the story in the [format specified above](#story-details).
+To make publishing stories easier, we provide a webhook that allows you to push stories to your CMS as they are completed. Our webhook will submit a POST request to the URL you specifiy so that you don't have to poll for newly completed stories. The body of the request will contain the JSON data for the story in the [format specified above](#story-details).
```

Comment:
Since this is dev documentation, I don't think we need to call out the actual button they need to click or give them the date of the change (i.e I think we can remove the note and update this paragraph). Maybe:

"To make publishing stories easier, we provide a webhook that allows you to push stories to your CMS after they are completed. When a user triggers the webhook on the Contently platform, it will submit a POST request to the URL you specify containing the JSON data for the story in the [format specified above](#story-details)."

@sanjayginde @finleye